### DISCUSSION TITLE
Should we release 0.73.4?

### DISCUSSION THREAD

**Discussion by blakef** [reactwg/react-native-releases#101] [Patches]
Current Release: [0.73.4](https://github.com/facebook/react-native/releases/tag/v0.73.4)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by dmytrorykun** [comment]
https://github.com/facebook/react-native/pull/42586 to fix https://github.com/facebook/react-native/issues/42477 (the `Unsupported top level event type "topAnimationFinish" dispatched` error).

**Reply by lunaleaps** [reply]
✅ picked

**Comment by cortinico** [comment]
I'd like to have this commit picked:
- https://github.com/facebook/react-native/commit/ca9b6b5038c419405a440fc8add7090be633654c

This suppresses a warning we're seeing with `react-native-screens` and Bridgeless Mode in 0.73 (cc @WoLewicki)

**Reply by lunaleaps** [reply]
✅ picked

**Comment by cipolleschi** [comment]
Please merge https://github.com/facebook/react-native/pull/42734 as it definitely fixes a concurrency issue and solve a runtime warning. The PR is already against 0.73 branch.

**Reply by lunaleaps** [reply]
✅ merged

**Comment by vksgautam1986** [comment]
@here  https://react-native-community.github.io/upgrade-helper/ is still showing 0.72.2 as latest version to update, can we update it to 0.73.3

**Reply by lunaleaps** [reply]
I'm guessing you mean 0.73.2 and when I go to the upgrade-helper now the latest version is 0.73.3

**Reply by vksgautam1986** [reply]
yeah sorry i means 0.73.2, now its working fine. thanks

**Reply by blakef** [reply]
Sorry, I fixed that. @lunaleaps should we be commenting on here when resolved?

**Comment by gituser8796** [comment]
Could you please include an important performance fix for Hermes JSON.stringify?

https://github.com/facebook/hermes/commit/8fd20f29af83ab59c5e21d7de8212bcc904dcc4a

https://github.com/reactwg/react-native-releases/discussions/98#discussioncomment-8057686

**Reply by lunaleaps** [reply]
See my comment on the disussion: https://github.com/reactwg/react-native-releases/discussions/98#discussioncomment-8363004

I don't think this should be going in

**Comment by lunaleaps** [comment]
Update: 

I have picked and merged all relevant changes for 0.73.4
Things we still need to do
1. Run `bump` as there are changes to `react-native-codegen`
2. Run two series of testing for 0.73.4

For 1., I'm not running bump yet as it actually publishes the package and its the weekend and I don't want to leave things in an intermediate state. I'll ping the release oncall folks and see who can volunteer to do the first round of testing. They can run the `bump`

For 2., I can run the second round of testing and release 0.73.4

Note: We should validate that this fix at minimum - https://github.com/facebook/react-native/pull/42734
